src/main/resources/assets/create/lang/ja_jp.json

```diff
@@ -1303,12 +1394,335 @@
+	"create.display_source.fill_level": "収納ブロックの使用率",
+	"create.display_source.fill_level.display": "表示形式",
+	"create.display_source.fill_level.percent": "パーセント",
+	"create.display_source.fill_level.progress_bar": "進行度バー",
+	"create.display_source.value_list.display": "数値の表示形式",
+	"create.display_source.value_list.shortened": "短縮",
+	"create.display_source.value_list.full_number": "完全な数値",
+	"create.display_source.value_list.thousand": "k",
+	"create.display_source.value_list.million": "m",
+	"create.display_source.player_deaths": "死亡回数",
+	"create.display_source.scoreboard": "スコアボード",
+	"create.display_source.scoreboard.objective": "Objective ID",
+	"create.display_source.scoreboard.objective.deaths": "死亡回数",
+	"create.display_source.time_of_day": "時刻",
+	"create.display_source.stop_watch": "ストップウォッチ",
+	"create.display_source.time.format": "時刻の表示形式",
+	"create.display_source.time.12_hour": "12時間制",
+	"create.display_source.time.24_hour": "24時間制",
+	"create.display_source.accumulate_items": "蓄積アイテムカウンター",
+	"create.display_source.item_throughput": "アイテム処理量",
+	"create.display_source.item_throughput.interval": "間隔",
+	"create.display_source.item_throughput.interval.second": "1秒あたり",
+	"create.display_source.item_throughput.interval.minute": "1分あたり",
+	"create.display_source.item_throughput.interval.hour": "1時間当たり",
+	"create.display_source.train_status": "鉄道運行の状態",
+	"create.display_source.station_summary": "鉄道駅の概要",
+	"create.display_source.station_summary.filter": "駅名フィルタ",
+	"create.display_source.station_summary.train_name_column": "列車列のサイズ",
+	"create.display_source.station_summary.platform_column": "乗降場列のサイズ",
+	"create.display_source.station_summary.now": "現在",
+	"create.display_source.station_summary.minutes": "分後",
+	"create.display_source.station_summary.seconds": "%1$s秒後",
+	"create.display_source.observed_train_name": "検知した列車名",
+	"create.display_source.max_enchant_level": "最大エンチャントレベル",
+	"create.display_source.boiler_status": "ボイラー状態",
+	"create.display_source.entity_name": "エンティティ名",
+	"create.display_source.kinetic_speed": "回転速度(RPM)",
+	"create.display_source.kinetic_speed.absolute": "回転方向を無視",
+	"create.display_source.kinetic_speed.directional": "回転方向を含める",
+	"create.display_source.kinetic_stress": "ネットワークの応力",
+	"create.display_source.kinetic_stress.display": "表示する情報",
+	"create.display_source.kinetic_stress.progress_bar": "進行度バー",
+	"create.display_source.kinetic_stress.percent": "パーセント",
+	"create.display_source.kinetic_stress.current": "使用しているSU",
+	"create.display_source.kinetic_stress.max": "合計の許容応力",
+	"create.display_source.kinetic_stress.remaining": "残りのSU",
+	"create.display_source.redstone_power": "レッドストーン強度",
+	"create.display_source.redstone_power.display": "表示形式",
+	"create.display_source.redstone_power.number": "数値",
+	"create.display_source.redstone_power.progress_bar": "進行度バー",
+	"create.display_source.boiler.not_enough_space": "スペースが足りません",
+	"create.display_source.boiler.for_boiler_status": " ボイラーステータス",
```

Comment:
Yes. Thanks for pointing that out.

Reply:
I don't think it necessarily needs to be based on the original text, as it will appear this way.

![](https://user-images.githubusercontent.com/77515401/183234980-149c20b5-8c61-4c1f-a817-eb2aeba6708c.jpg)